quickstarts/aws/amazon-route53/config.yml

```diff
@@ -0,0 +1,39 @@
+id: bf043e5a-213c-411e-8eea-4e386a29a088
```

Comment:
For new quickstarts, we will generate an ID for you automatically when the PR is merged. We can remove this line:
```suggestion
```